commit/trk2dictionary/trk2dictionary.pyx

```diff
@@ -80,7 +80,7 @@ cpdef run( filename_trk, path_out, filename_peaks = None, filename_mask = None,
     # fiber-tracts from .trk
     print '\t\t* tractogram'
     try :
-        fib, trk_hdr = nibabel.trackvis.read( filename_trk )
+        _, trk_hdr = nibabel.trackvis.read( filename_trk, as_generator=True )
```

Comment:
I see that in the next call when we actually load the tracts. But in this call, we simply load the header. If so, perhaps we don't need to specify `as_generator=True`, correct?